### PR TITLE
Copyediting, part 2

### PR DESCRIPTION
**Pull request by mjingle**
Signed-off-by: Mary Jinglewski <mjinglewski@chef.io>

Copyediting for Effortless docs, part 1. Some comments are included for the purposes of clarifying details.

### CODE REVIEW

File: effortless-chef-io/effortless-config.md [hunk 1]
```diff
@@ -4,43 +4,51 @@ Effortless Config is the pattern for managing your Chef Infra workloads. It uses
...
    pkg_scaffolding="chef/scaffolding-chef-infra"
    pkg_svc_user=("root")
-   scaffold_policy_name="<YOUR POLICYFILE NAME>"
+   scaffold_policy_name="<MY POLICYFILE NAME>"
    ```
-1. Create a policyfile directory in your chef-repo and build a policyfile
-   
-   Example of a policyfile.rb:
+
+1. Create a Policyfile directory in the chef-repo and build a Policyfile
```
Comment (mjingle) [inline effortless-chef-io/effortless-config.md:46]: @TheLunaticScripter Is there a specific command required to build a Policyfile?

File: effortless-chef-io/effortless-config.md [hunk 2]
```diff
@@ -4,43 +4,51 @@ Effortless Config is the pattern for managing your Chef Infra workloads. It uses
 
 ## Patterns
 
-### Chef Repo Cookbook pattern
+### Chef Repo Cookbook Pattern
 
-This pattern is used to build policyfiles using the chef_repo pattern for organizing your cookbooks. The chef-repo pattern can be found [here](https://docs.chef.io/chef_repo/).
+This pattern builds Policyfiles using the chef-repo pattern for organizing your cookbooks. More information about the [chef-repo](https://docs.chef.io/chef_repo/) pattern can be found in our Chef Infra Documentation. View an example of this pattern [here](https://github.com/chef/effortless/tree/master/examples/effortless_config/chef_repo_pattern).
```
Comment (susanev) [inline effortless-chef-io/effortless-config.md:9]:
```suggestion
This pattern builds Policyfiles using the [chef-repo pattern](https://github.com/chef/effortless/tree/master/examples/effortless_config/chef_repo_pattern) for organizing your cookbooks. Our Chef Infra documentation contains more information about the [chef-repo](https://docs.chef.io/chef_repo/) pattern.
```

or something else that doesn't have a here link

File: effortless-chef-io/effortless-config.md [hunk 3]
```diff
@@ -134,55 +150,80 @@ This pattern is used to build policyfiles using the chef_repo pattern for organi
...
+1. To run the Policyfile on a system, install Chef Habitat services and run:
+
+   ```bash
+   hab svc load <my_origin>/<my_policyfile_name>
+   ```
 
 ### Policyfile Cookbook pattern
 
-This is a pattern of building an artifact for a single cookbook. 
+This pattern builds an artifact for a single cookbook. View an example of this pattern [here](https://github.com/chef/effortless/tree/master/examples/effortless_config/policyfile_cookbook_pattern).
```
Comment (susanev) [inline effortless-chef-io/effortless-config.md:185]:
```suggestion
The [cookbook pattern]((https://github.com/chef/effortless/tree/master/examples/effortless_config/policyfile_cookbook_pattern)) builds an artifact for a single cookbook.
```

Comment (kagarmoe) [reply effortless-chef-io/effortless-config.md:185]: We need to be clear that we're providing an example Policyfile cookbook. Cookbooks vary.

File: effortless-chef-io/effortless-config.md (hunk 1 above)
Comment (TheLunaticScripter) [reply effortless-chef-io/effortless-config.md:46]: I totally didn't realize but it looks like `chef generate policyfile`  creates a `Policyfile.rb` in the directory that the command is run in.

File: effortless-chef-io/effortless-config.md (hunk 2 above)
Comment (TheLunaticScripter) [reply effortless-chef-io/effortless-config.md:9]: Okay that makes sense.

File: effortless-chef-io/effortless-config.md (hunk 1 above)
Comment (mjingle) [reply effortless-chef-io/effortless-config.md:46]: Oh cool! Thanks for that! So in "building" a Policyfile, do you mean that you're adding content to the empty Policyfile?

Comment (TheLunaticScripter) [reply effortless-chef-io/effortless-config.md:46]: Yes createing the file and adding the content.

Comment (collinmcneese) [reply effortless-chef-io/effortless-config.md:46]: We could reference the creation if we wanted as well:

```shell
mkdir policyfiles
chef generate policyfile policyfiles/MyPolicyfile.rb                                                                                
Recipe: code_generator::policyfile
  * template[/workspace/my_repo/policyfiles/MyPolicyfile.rb] action create
    - create new file /workspace/my_repo/policyfiles/MyPolicyfile.rb
    - update content in file /workspace/my_repo/policyfiles/MyPolicyfile.rb from none to fc117f
    (diff output suppressed by config)
```

**Comment by kagarmoe** [comment]
Agreed--its best to use named links!

File: effortless-chef-io/effortless-config.md [hunk 4]
```diff
@@ -134,55 +150,80 @@ This pattern is used to build policyfiles using the chef_repo pattern for organi
...
    ```
-1. To run your policyfile on a system you just need to install habitat as a service and run `hab svc load <your_origin>/<your_policyfile_name>`
+
+1. To run the Policyfile on a system, install Chef Habitat services and run:
+
+   ```bash
+   hab svc load <my_origin>/<my_policyfile_name>
+   ```
 
 ### Policyfile Cookbook pattern
```
Comment (kagarmoe) [inline effortless-chef-io/effortless-config.md:183]:
```suggestion
### Policyfile Cookbook Pattern
```

File: effortless-chef-io/effortless-config.md (hunk 2 above)
Comment (kagarmoe) [reply effortless-chef-io/effortless-config.md:9]:
```suggestion
This pattern uses the [chef-repo](https://docs.chef.io/chef_repo/) to store and organize everything you need to define your infrastructure with Chef Infra, including:

- Cookbooks (including recipes, attributes, custom resources, libraries, and templates)
- Data bags
- Policyfiles

The Effortless Config pattern builds Policyfiles using the chef-repo. View an [example chef-repo](https://github.com/chef/effortless/tree/master/examples/effortless_config/chef_repo_pattern) in the Chef Effortless GitHub repository.
```

File: effortless-chef-io/effortless-config.md [hunk 5]
```diff
@@ -4,43 +4,51 @@ Effortless Config is the pattern for managing your Chef Infra workloads. It uses
...
+   ```bash
    mkdir habitat
    ```
+
 1. Make a plan file
-   
-   > Notes: For a cookbook targeting windows use a `plan.ps1` for Linux use a `plan.sh` if your cookbook targets both windows and linux you can have both a `plan.ps1` and a `plan.sh` in your habitat directory.
-   ```
+
+   Use a `plan.ps1` for a cookbook targeting Windows. Use a `plan.sh` for a cookbook targeting Linux. If the cookbook targets both Windows and Linux, you can have both a `plan.ps1` and a `plan.sh` in the `habitat` directory. Create a plan in Linux with the following command:
```
Comment (kagarmoe) [inline effortless-chef-io/effortless-config.md:25]:
```suggestion
   Make a `plan.ps1` for a cookbook that targets Microsoft Windows systems and a `plan.sh` for a cookbook that targets Linux systems. You can have both a `plan.ps1` and a `plan.sh` in the `habitat` directory for cookbooks that target both systems. Create a plan in Linux with the following command:
```

File: effortless-chef-io/effortless-config.md (hunk 1 above)
Comment (kagarmoe) [reply effortless-chef-io/effortless-config.md:46]:
```suggestion
1. If you do not have a policyfiles directory in your chef-repo, create one:
 
   ```
   mkdir policyfiles
   ```

1. Generate a Policyfile

  ```
  chef generate policyfile policyfiles/my_policyfile
  ```
```

Comment (kagarmoe) [reply effortless-chef-io/effortless-config.md:46]: Does chef generate `my_policyfile` require the `.rb` extension?

File: effortless-chef-io/effortless-config.md [hunk 6]
```diff
@@ -87,13 +101,15 @@ This pattern is used to build policyfiles using the chef_repo pattern for organi
    suites:
      - name: base
        provisioner:
-         arguments: ["<YOUR ORIGIN>", "<YOUR PACKAGE NAME>"]
+         arguments: ["<MY ORIGIN>", "<MY PACKAGE NAME>"]
```
Comment (kagarmoe) [inline effortless-chef-io/effortless-config.md:104]:
```suggestion
         arguments: ["<MY_ORIGIN>", "<MY_PACKAGE_NAME>"]
```

File: effortless-chef-io/effortless-config.md [hunk 7]
```diff
@@ -125,7 +141,7 @@ This pattern is used to build policyfiles using the chef_repo pattern for organi
    echo "Installing $latest_hart_file"
    hab pkg install $latest_hart_file
 
-   echo "Determing pkg_prefix for $latest_hart_file"
+   echo "Determining pkg_prefix for $latest_hart_file"
```
Comment (kagarmoe) [inline effortless-chef-io/effortless-config.md:144]: @TheLunaticScripter Can we check it this typo is in the code?

File: effortless-chef-io/effortless-config.md (hunk 3 above)
Comment (kagarmoe) [reply effortless-chef-io/effortless-config.md:185]:
```suggestion
This pattern builds a Chef Habitat artifact for the Policyfile cookbook. You can find an [example Policyfile cookbook](https://github.com/chef/effortless/tree/master/examples/effortless_config/policyfile_cookbook_pattern) in the Chef Effortless GitHub repository.
```

File: effortless-chef-io/effortless-config.md [hunk 8]
```diff
@@ -224,13 +269,15 @@ This is a pattern of building an artifact for a single cookbook.
    suites:
    - name: base
       provisioner:
-         arguments: ["<YOUR ORIGIN>", "<YOUR COOKBOOK NAME>"]
+         arguments: ["<MY ORIGIN>", "<MY COOKBOOK NAME>"]
```
Comment (kagarmoe) [inline effortless-chef-io/effortless-config.md:272]:
```suggestion
         arguments: ["<MY_ORIGIN>", "<MY_COOKBOOK_NAME>"]
```

File: effortless-chef-io/effortless-config.md (hunk 7 above)
Comment (TheLunaticScripter) [reply effortless-chef-io/effortless-config.md:144]: yep I'll take a look.

File: effortless-chef-io/effortless-config.md (hunk 1 above)
Comment (TheLunaticScripter) [reply effortless-chef-io/effortless-config.md:46]: it does not require the `.rb` extension

**Comment by susanev** [comment]
my gray checkmark is sad, but thanks for making the link text much much better 🎉

File: effortless-chef-io/effortless-config.md [hunk 9]
```diff
@@ -134,55 +166,80 @@ This pattern is used to build policyfiles using the chef_repo pattern for organi
...
-   pkg_origin=<your Origin>
+
+   ```bash
+   pkg_name=<Name of my cookbook artifact>
+   pkg_origin=<My Origin>
    pkg_version="<Cookbook version>"
-   pkg_maintainer="<Your Name>"
-   pkg_license=("<License for you cokbook example Apache-2.0>")
+   pkg_maintainer="<My Name>"
+   pkg_license=("<License for my cookbook example Apache-2.0>")
```
Comment (kagarmoe) [inline effortless-chef-io/effortless-config.md:232]:
```suggestion
   pkg_license=("<License for my_cookbook example Apache-2.0>")
```

File: effortless-chef-io/effortless-config.md [hunk 10]
```diff
@@ -200,12 +257,16 @@ This is a pattern of building an artifact for a single cookbook.
...
-1. Build the package
-   ```
-   hab pkg build <Your Cookbook Name>
-   ```
-1. Modify your `kitchen.yml` file to look like this
+
+1. Build the package:
+
+   ```bash
+   hab pkg build <My Cookbook Name>
```
Comment (kagarmoe) [inline effortless-chef-io/effortless-config.md:264]:
```suggestion
   hab pkg build <my_cookbook>
```

File: effortless-chef-io/effortless-config.md [hunk 11]
```diff
@@ -224,13 +285,15 @@ This is a pattern of building an artifact for a single cookbook.
    suites:
    - name: base
       provisioner:
-         arguments: ["<YOUR ORIGIN>", "<YOUR COOKBOOK NAME>"]
+         arguments: ["<MY_ORIGIN>", "<MY_COOKBOOK_NAME>"]
```
Comment (kagarmoe) [inline effortless-chef-io/effortless-config.md:288]:
```suggestion
         arguments: ["<my_origin>", "<my_cookbook>"]
```

File: effortless-chef-io/effortless-config.md [hunk 12]
```diff
@@ -4,43 +4,67 @@ Effortless Config is the pattern for managing your Chef Infra workloads. It uses
...
+   pkg_name=<NAME FOR MY POLICYFILE>
+   pkg_origin=<MY ORIGIN>
    pkg_version="0.1.0"
-   pkg_maintainer="YOUR NAME AND EMAIL"
+   pkg_maintainer="MY NAME AND EMAIL"
    pkg_license=("Apache-2.0")
    pkg_scaffolding="chef/scaffolding-chef-infra"
    pkg_svc_user=("root")
-   scaffold_policy_name="<YOUR POLICYFILE NAME>"
+   scaffold_policy_name="<MY POLICYFILE NAME>"
```
Comment (mjingle) [inline effortless-chef-io/effortless-config.md:49]: @kagarmoe What about this one? :)

Comment (mjingle) [reply effortless-chef-io/effortless-config.md:49]: There's another on line 43

Comment (kagarmoe) [reply effortless-chef-io/effortless-config.md:49]: Good catch! `my_policyfile` because we can't have spaces in the policyfile name

Comment (mjingle) [reply effortless-chef-io/effortless-config.md:49]: And it goes with the example you put further down. BRB gonna update the other PR, too.

**Pull request merged**